src/test/kotlin/org/rust/ide/navigation/goto/RsGotoSuperHandlerTest.kt

```diff
@@ -139,6 +139,19 @@ class RsGotoSuperHandlerTest : RsTestBase() {
             mod bar;
     """)
 
+    // Tried to mock Cargo this way but it didn't work (it does work when running IntellJ). Any tips?
```

Comment:
Hmm, but this test passes on CI. Chat's wring here?

Reply:
@vlad20012 This test didn't pass - it just wasn't run because of `-` at the beginning of the method name